### DISCUSSION TITLE
How to Replicate scenes in a real environment.

### DISCUSSION THREAD

**Discussion by dhgarcia** [allenai/ai2thor#749] [General]
The paper [RoboTHOR: An Open Simulation-to-Real Embodied AI Platform](https://arxiv.org/pdf/2004.06799.pdf)  talks about replicability 

> Replicable - The physical space has been designed to be easily replicable by other researchers should they wish to construct their own physical environment. This is achieved through open sourced plans, readily available building components, IKEA furniture, and a low cost amounting to roughly $10,000 in materials and assets to create the physical space.

Is there a source for more information on this?
Are the plans open sourced yet?
We would be interested in building also a similar space, and wouldn't to know more on what that is in the implementation.

**Comment by roozbehm** [comment]
Unfortunately, due to COVID, we had to shut down the lab for RoboTHOR since nobody could attend the office. We are planning to re-open it soon. We will run the experiments for teams that achieve high performance in simulation. 

We can share the layouts and the list of items if you are planning to reconstruct the scenes.
